### PR TITLE
Fix #27585: Add firewall-cmd --reload

### LINKED ISSUE #27585: [enterprise-4.6] Issue in file installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
### Which section(s) is the issue in?

In **Creating an RHCOS images cache (optional)**, **2. Open firewall port 8080 to be used for RHCOS Image caching.**

### What needs fixing?

`firewalld` should be reloaded. Add `firewall-cmd --reload`:

```
[kni@provisioner ~]$ sudo firewall-cmd --add-port=8080/tcp --zone=public --permanent
[kni@provisioner ~]$ sudo firewall-cmd --reload
```

### PR DESCRIPTION
**Pull request by kurokobo**
Add `firewall-cmd --reload` just after `--add-port` command to apply the change.

### CODE REVIEW

**Comment by kurokobo** [comment]
@vikram-redhat Please review this PR.

**Comment by vikram-redhat** [comment]
@johnwilkins can you ack?

@ahardin-rh for peer review afterwards.

**Review by iranzo — APPROVED** [review]

**Comment by rlopez133** [comment]
LGTM

**Comment by ahardin-rh** [comment]
LGTM

**Comment by ahardin-rh** [comment]
I reached out to Amit Ugol for QE review.

**Comment by tradej** [comment]
Tracked in [TELCODOCS-219](https://issues.redhat.com/browse/TELCODOCS-219).

**Comment by ktothill** [comment]
Reviewing PR with John Wilkins, as{{ --Reload}} command may not be the current best practice. May need to rework commands as: 

[kni@provisioner ~]$ sudo firewall-cmd --add-port=8080/tcp --zone=public
[kni@provisioner ~]$ sudo firewall-cmd --permanent

LGTM. Confirmed with John that --Reload command should be fine to use.

**Comment by kurokobo** [comment]
Thank you for the review. I'm sorry for the delay, but I've added a commit that resolves the conflict.

**Comment by tradej** [comment]
Thank you @kurokobo. QE will review this PR likely next week.

**Comment by tradej** [comment]
We are currently talking to QE about verification requirements, sorry for the delay.

**Review by awolffredhat — APPROVED** [review]
Approved by Lubov and Raviv

**Review by awolffredhat — APPROVED** [review]

File: modules/ipi-install-creating-an-rhcos-images-cache.adoc [hunk 1]
```diff
@@ -24,6 +24,7 @@ $ sudo dnf install -y podman
 [source,terminal]
 ----
 $ sudo firewall-cmd --add-port=8080/tcp --zone=public --permanent
+$ sudo firewall-cmd --reload
```
Comment (bergerhoffer) [inline modules/ipi-install-creating-an-rhcos-images-cache.adoc:27]: There shouldn't be more than one command in a code block (per our guidelines here: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output)

Do you mind updating to put this command in a separate code block?

**Review by bergerhoffer — COMMENTED** [review]
One thing, and then the commits in the PR need will need to be squashed to 1 commit.

File: modules/ipi-install-creating-an-rhcos-images-cache.adoc (hunk 1 above)
Comment (kurokobo) [reply modules/ipi-install-creating-an-rhcos-images-cache.adoc:31]: Added `sudo firewall-cmd --reload` as a new code block.

**Comment by kurokobo** [comment]
Thanks for reviewing, fixed and force-pushed into 1 commit.

**Comment by bergerhoffer** [comment]
Thanks @kurokobo! And I only just now realized that the rest of the file and assembly incorrectly has multiple commands in a single code block. I appreciate you fixing it to do it the right way here though for this new addition. At some point, someone should come back through this file and fix those. For another day :) 

Thanks, merging!

**Pull request merged**